README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Athens Banner](./docs/static/banner.png)
 
-[![Build Status](https://cloud.drone.io/api/badges/gomods/athens/status.svg)](https://cloud.drone.io/gomods/athens)
+[![Build Status](https://github.com/gomods/athens/actions/workflows/ci.yml/badge.svg)](https://github.com/gomods/athens/actions/workflows/ci.yml)
```

Comment:
Hyperlink should ideally lead to the build page, probably: https://github.com/gomods/athens/actions